angularD3/locale/index.coffee

```diff
@@ -0,0 +1,26 @@
+localeDefinitions = {}
+
+localeDefinitions["ca-es"] = require("./ca-ES")
+localeDefinitions["de-ch"] = require("./de-CH")
+localeDefinitions["de-de"] = require("./de-DE")
+localeDefinitions["en-ca"] = require("./en-CA")
+localeDefinitions["en-gb"] = require("./en-GB")
+localeDefinitions["en-us"] = require("./en-US")
+localeDefinitions["en-es"] = require("./es-ES")
+localeDefinitions["fi-fi"] = require("./fi-FI")
+localeDefinitions["fr-ca"] = require("./fr-CA")
+localeDefinitions["fr-fr"] = require("./fr-FR")
+localeDefinitions["he-il"] = require("./he-IL")
+localeDefinitions["hu-hu"] = require("./hu-HU")
+localeDefinitions["it-it"] = require("./it-IT")
+localeDefinitions["ja-jp"] = require("./ja-JP")
+localeDefinitions["ko-kr"] = require("./ko-KR")
+localeDefinitions["mk-mk"] = require("./mk-MK")
+localeDefinitions["nl-nl"] = require("./nl-NL")
+localeDefinitions["pl-pl"] = require("./pl-PL")
+localeDefinitions["pt-br"] = require("./pt-BR")
+localeDefinitions["ru-ru"] = require("./ru-RU")
+localeDefinitions["sv-se"] = require("./sv-SE")
+localeDefinitions["zh-cn"] = require("./zh-CN")
```

Comment:
We should make it possible to not require every single locale definition file. Just the ones you need to use.